proto/xudrpc.proto

```diff
@@ -455,16 +464,19 @@ message LndChannels {
   uint32 inactive = 2 [json_name = "inactive"];
   // The number of channels that are pending on-chain confirmation before they can be used.
   uint32 pending = 3 [json_name = "pending"];
+  // The number of channels that are pending on-chain confirmation before they can be used.
+  uint32 closed = 4 [json_name = "closed"];
 }
 
 message LndInfo {
-  string error = 1 [json_name = "error"];
-  LndChannels channels = 2 [json_name = "channels"];
-  repeated Chain chains = 3 [json_name = "chains"];
-  uint32 blockheight = 4 [json_name = "blockheight"];
-  repeated string uris = 5 [json_name = "uris"];
-  string version = 6 [json_name = "version"];
-  string alias = 7 [json_name = "alias"];
+  SwapClientStatus status = 1 [json_name = "status"];
```

Comment:
Same here for renumbering all these fields.

Reply:
At first I thought we want the requested order, but you're right.